org.kde.ktuberling.appdata.xml

```diff
@@ -0,0 +1,145 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>org.kde.ktuberling.desktop</id>
```

Comment:
Please don't duplicate appstream data. Is there anything wrong with the one upstream? If there is, patches very welcome.

Reply:
The appstream was only added six days ago! It's not in this release ;-)

I'll absolutely send a patch for the OARS and version tags